src/Packetery/Module/Order/Repository.php

```diff
@@ -487,9 +487,9 @@ public function getByIds( array $orderIds ): array {
 	 * @param int   $maxDays               Max number of days of single packet sync.
 	 * @param int   $limit                 Number of records.
 	 *
-	 * @return iterable|Order[]
+	 * @return int[]
```

Comment:
Nebylo by vhodnější Iterator<int> ?

Reply:
Proč používat interface, když víme, že to vrací pole integerů?

Reply:
Reálně to je Generator, upraveno.